site/content/operational_guide/resource_limits.md

```diff
@@ -82,6 +92,18 @@ limits:
     # and read until the lookback period resets.
     lookback: 15s
 
+  # If set, will enforce a maximum cap on the bytes read from disk that make up time series objects themselves (not their data).
+  # This limit can be used to ensure queries that match an extremely high volume of series can be limited before even
+  # reading the underlying series data from disk.
+  maxRecentlyQueriedSeriesDiskRead:
+    # Value sets the maximum disk bytes read to make up the time series objects.
```

Comment:
hmm the unit isn't bytes here right? It's # of series?

Same applies to the intro comment above